test/over_react_redux/connect_flux_test.dart

```diff
@@ -446,8 +455,11 @@ main() {
             await Future(() {});
 
             // store.state.count should be 1 but does not re-render due to override in `areStatePropsEqual`
-            expect(methodsCalled.map((methodObj) => methodObj['called']),
-                updateMethodCalls);
+            expect(
+                methodsCalled,
+                expectedUpdateMethodCalls
+                    .map((expected) => containsPair('called', expected))
+                    .toList());
```

Comment:
This pattern results in the whole calls being included in the test failure messages, which provides a lot of helpful contextual info.

Before:
```
  Expected: ['mapStateToProps',  'areStatePropsEqual']
    Actual: ['mapStateToProps', 'areStatePropsEqual', 'mapStateToProps', 'areStatePropsEqual']
```
With these changes:
```
  Expected: [
              <contains pair 'called' => 'mapStateToProps'>,
              <contains pair 'called' => 'areStatePropsEqual'>
            ]
    Actual: [
              {
                'called': 'mapStateToProps',
                'state': CounterState:CounterState:{count: 1, name: Counter}
              },
              {
                'called': 'areStatePropsEqual',
                'prev': {'ConnectFluxCounterProps.currentCount': 0},
                'next': {'ConnectFluxCounterProps.currentCount': 1}
              },
              {
                'called': 'mapStateToProps',
                'state': CounterState:CounterState:{count: 1, name: Counter}
              },
              {
                'called': 'areStatePropsEqual',
                'prev': {'ConnectFluxCounterProps.currentCount': 1},
                'next': {'ConnectFluxCounterProps.currentCount': 1}
              }
            ]
```